### ISSUE TITLE
Deprecation warning in Django 2.1.1

### ISSUE THREAD

**Issue by harrybiddle** [hzdg/django-enumfields#87]
Upon upgrading to Django 2.1.1, I'm getting a lot of this:

```
/app/.heroku/python/lib/python3.6/site-packages/django/db/models/sql/compiler.py:995: RemovedInDjango30Warning:
Remove the context parameter from EnumField.from_db_value(). Support for it will be removed in Django 3.0.
```

I'm guessing this is due to enumfields/fields.py:72:

```python
    def from_db_value(self, value, expression, connection, context):
        return self.to_python(value)
```

Is there a way we can drop the last parameter, dependent on the version of Django being used, or possibly just convert this to `*args`?

**Comment by harrybiddle** [comment]
Will be closed by #85

**Comment by johnthagen** [comment]
#85 has been merged.
